test/e2e/persistent_volumes-disruptive.go

```diff
@@ -107,12 +97,8 @@ var _ = framework.KubeDescribe("PersistentVolumes [Volume][Disruptive]", func()
 		}
 	})
 
-	AddCleanupAction(func() {
```

Comment:
Should L93 "Expect(clientNodeIP).NotTo(BeEmpty())" be moved to L96.1? That way you ensure that you have a non-empty ip in the case of the for loop completing with no match. It is an unusual case but is simple to catch.